WordPress/src/main/java/org/wordpress/android/util/config/BloggingPromptsFeature.kt

```diff
@@ -0,0 +1,10 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import javax.inject.Inject
+
+class BloggingPromptsFeature @Inject constructor() {
```

Comment:
❓ I am a bit unsure about making this a regular class instead of still being a Feature Flag. I understand we should get rid of feature flags when we are not using them anymore, but I wonder if it would make sense for this to still be a regular `FeatureConfig` that can be remotely enabled/disabled. Wdyt?

My reason is that I personally think there are 2 kinds of FFs:
1. Longstanding Feature Flags, which let us show/hide a whole feature based on a remote config. Those are useful to hide features when they malfunction due to external causes (e.g.: API issues) and also when deprecating/deleting features that are already in production (e.g.: even users on older versions would have the feature removed).
2. Development/Rollout/Rollback Feature Flags, which are used during development so we can merge incomplete work in the trunk and enable it when we are ready to ship/rollout (or do A/B testing), and also act as a safeguard to rollback a feature if we see issues when it's deployed.

I agree the second kind should be temporary, but I think it might be good to have some of the first kind as well. Not 100% sure though.

Reply:
Hey @thomashorta 
I am unsure about this particular feature flag, but I think removing the feature flag once the feature is in production for 2-3 releases is a good practice. We now have around 50 feature flags in the app, which feels too much. I feel that removing the ones that are not in use would make the code and testing much more straightforward. 

>Longstanding Feature Flags let us show/hide a whole feature based on a remote config. Those are useful to hide features when they malfunction due to external causes (e.g.: API issues) 

Longstanding Feature Flags make sense in the scenario of issues arising due to external causes, but I don't think in the case of Blogging Prompt, I don't think there are any, So I would suggest removing this feature flag. 
 

>and also when deprecating/deleting features that are already in production (e.g.: even users on older versions would have the feature removed)

I fear if we consider this case, we will never remove feature flags 😅 . I don't recall an instance in WordPress where we had to turn off a feature after 2-3 releases, so this is a rare case. 

Having said all this, @thomashorta, if you feel 💯 about keeping this feature flag, you can revert the changes I have made for the Blogging prompt feature flag 👍🏼

Reply:
> I don't recall an instance in WordPress where we had to turn off a feature after 2-3 releases, so this is a rare case.

Yeah, I agree with you. I think it's safe to remove it.